src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -74,7 +74,7 @@ abstract class EventShape extends mxEllipse {
   // This will be removed after implementation of all supported events
   private markNonFullyRenderedEvents(c: mxAbstractCanvas2D): void {
     const eventKind = StyleUtils.getBpmnEventKind(this.style);
-    if (eventKind == ShapeBpmnEventKind.CANCEL) {
```

Comment:
Can you let the Cancel event render, and just add an other color and check for the Conditional event ?

So we can merge this PR, even if the PR #789 is not yet approved 😃